com.unity.formats.fbx/Editor/FbxPropertyChannelPair.cs

```diff
@@ -118,6 +118,15 @@ public PropertyChannelMap(Dictionary<string,string> propertyMap, Dictionary<stri
                 { @"m_RotationOffsets\.Array\.data\[(\d+)\]", "{0}.Offset R" }
             };
 
+        /// <summary>
+        /// Map of Unity blendshape property name as a regular expression to the FBX property.
+        /// This is necessary because the Unity property contains the name of the target object.
+        /// </summary>
+        private static Dictionary<string, string> MapBlendshapesPropToFbxProp = new Dictionary<string, string>()
```

Comment:
Tech-debt to log in jira: these dictionaries should all be lists. Initially it was envisioned as a map from the Unity name to the FBX name, but now they regex subs and can't be searched, just iterated. So the overhead of using dictionaries is wasted -- and it means the order that we iterate is implementation-dependent.

Reply:
On reading more: not all should be lists, just some.

Reply:
Not sure how tech debt stories are to be titled (whether a user persona is to be used), but I've logged the story here: https://jira.unity3d.com/browse/FBX-95